src/Sql/Sql.Sdk/Generated/Models/ServerBlobAuditingPolicy.cs

```diff
@@ -293,7 +293,7 @@ public ServerBlobAuditingPolicy(BlobAuditingPolicyState state, string id = defau
         /// audit logs.
         ///
         /// For more information, see [Database-Level Audit Action
-        /// Groups](https://learn.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups).
+        /// Groups](https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups).
```

Comment:
The docs URL has been updated to learn.microsoft.com instead of docs.microsoft.com. Please revert the change and re-generate the code.